src/content/docs/dev/amplifier/chain-integration/relay-messages/manual.mdx

```diff
@@ -83,15 +191,56 @@ axelard tx wasm execute $SOURCE_CHAIN_GATEWAY \
     --chain-id axelar-testnet-lisbon-3 \
     --node $RPC
 ```
+</tab-item>
+</tabs>
 
 ## Construct a proof
-
+Now that the message has been routed, the [Prover](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/doc/src/contracts/multisig_prover.md) of the destination chain can [construct the proof](https://devnet-amplifier.axelarscan.io/tx/6693F9316263A079ECC75664E843BBF91C9F7F0B6A5723102886AF57954C9516) for it.
 [`construct_proof`](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/contracts/multisig-prover/src/msg.rs#L61) takes a vector of `CrossChainID`s and builds a proof that includes the messages to be routed so that the message can be relayed to the destination gateway. It also passes the message from the gateway to the prover.
 
+<tabs>
+<tab-item title="devnet">
+```bash
+export DESTINATION_CHAIN_MULTISIG_PROVER="axelar14smpvv72hkaajh7rqzuhxum5vj3yh5smftaescucdmfsh9eednyqdme35j" # Chain integrations unique prover
+export CHAIN_NAME="toronto" # Chain name
+export TX_HASH="0x4f5b0ed0dc717b317f16c841a407c79226deba7bc15e82ba98b4d4aab1169baf" # Hash of tx on Toronto chain
+export TX_EVENT="1" # Index of call-contract event on Toronto chain
+export RPC="http://devnet-amplifier.axelar.dev:26657" # Devnet RPC
+```
+</tab-item>
+<tab-item title="testnet">
 ```bash
 export DESTINATION_CHAIN_MULTISIG_PROVER="axelar1xz4cya4qm2ws6nzperhvc40wdjcq4872fl6d3j2s4cytyx8j80eqenv87g"
+export CHAIN_NAME="avalanche"
+export TX_HASH="0x41d08bd627d7be301e0858312b63b3ede840d7fad145a9da333c26adf5d98614"
+export TX_EVENT="0"
+export RPC="https://tm.axelar-testnet.lava.build:443"
 ```
+</tab-item>
+</tabs>
+
+<tabs>
```

Comment:
Why tabs when the command is identical? commands should be fixed, variable defaults should change by environment.